docs/quickstart/quickstart_code.py

```diff
@@ -55,6 +54,13 @@ def from_country(self, country: str) -> sqlalchemy.ColumnElement:
         """
         return Candidate.country == country
 
+    @decorators.view_aggregation()
+    def count_by_column(self, filtered_query: sqlalchemy.Select, column_name: str) -> sqlalchemy.Select:  # pylint: disable=W0602, C0116, W9011
```

Comment:
For me it's too general, besides, as I understand we will print the whole signature to the LLM, how is it supposed to fill the `filter_query` argument? Also `column_name` will be a hallucination without proper context. I'd change it to something more specific and update the signature accordingly. `filtered_query` should be accessible via `self`.

Reply:
`filtered_query` is a hidden argument. Please see: https://github.com/deepsense-ai/db-ally/blob/pw/add-single-aggregation/src/dbally/views/methods_base.py#L18

Reply:
ok, but why its here, if it can be accessed via `self`?

Reply:
https://github.com/deepsense-ai/db-ally/pull/62/files/2918ba5a81d0da7e3247964b5d792f3a7c1ee080#diff-c9a06e39afec54ec8db60598f66db7326f0ff226303b29e8d7e211960a77761bR130
See how this parameter is injected.